### PR TITLE
Migrate elixir install

### PR DESCRIPTION
**Pull request by mwu3535**
# Summary

Migrate Elixir install.

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by brammerl** [comment]
@mwu3535 looks good a couple things: 
1. Can you sign the CLA please? 
2. Is there a timeline for when you want this merged? I notice it's not behind a feature flag so the moment we merge this PR, the changes will take affect and show the new install experience to all users in production.

File: data-sources/elixir/config.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ displayName: Elixir agent installation
 description: Start monitoring the performance of an Elixir application or service by installing the New Relic Elixir agent.
 install:
   primary:
-    link:
-      url: https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/elixir/elixir-open-source-agent
+    type: nerdlet
+    nerdlet_id: marketplace.install-data-source
+    nerdlet_state:
+      dataSourceId: elixir
+    requires_account: false
```
Comment (brammerl) [inline data-sources/elixir/config.yml:10]:
```suggestion
    nerdlet: 
    nerdletId: marketplace.install-data-source
    nerdletState:
      dataSourceId: elixir
    requiresAccount: false
```

Comment (brammerl) [reply data-sources/elixir/config.yml:10]: Here is how you format data source nerdlet navigation: https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-source-install-fields

it is different than the way we format the internal data sources.

File: data-sources/elixir/config.yml [hunk 2]
```diff
@@ -3,6 +3,9 @@ displayName: Elixir agent installation
 description: Start monitoring the performance of an Elixir application or service by installing the New Relic Elixir agent.
 install:
   primary:
-    link:
-      url: https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/elixir/elixir-open-source-agent
+    nerdlet: 
+      nerdletId: marketplace.install-data-source
+      nerdletState:
+       dataSourceId: elixir
```
Comment (aswanson-nr) [inline data-sources/elixir/config.yml:9]:
```suggestion
      nerdletState:
        dataSourceId: elixir
```

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**